api/questions.js

```diff
@@ -11,17 +11,16 @@ const { requireQueue, requireQuestion, failIfErrors } = require('./util')
 const requireCourseStaffForQueueForQuestion = require('../middleware/requireCourseStaffForQueueForQuestion')
 
 
-async function modifyBeingAnswered(questionId, answering) {
-  const question = await Question.findOne({ where: { id: questionId } })
+/* eslint-disable no-param-reassign */
+function modifyBeingAnswered(question, answering) {
   question.beingAnswered = answering
   if (answering) {
     question.answerStartTime = new Date()
   } else {
     question.answerEndTime = new Date()
   }
-
-  return question.save()
```

Comment:
why don't you need a save here?

Reply:
This makes the function a bit more specific in its functionality. Take the `POST /` route: I also want to modify the `answeredById` attribute before saving.

Reply:
ahhhh gotcha :)